--- ies.orig/Makefile	2017-07-03 15:51:48.361221142 +0200
+++ ies/Makefile	2017-07-03 15:54:45.439368367 +0200
@@ -83,6 +83,11 @@
 	@+$(call kernelbuild,clean)
 	@-rm -rf *.${MANSECTION}.gz *.ko
 
+# Pure module install
+modules_install: default
+	@echo "Installing modules..."
+	@+$(call kernelbuild,modules_install)
+
 # Install the modules and manpage
 install: default manfile
 	@echo "Copying manpages..."
