--- files/Makefile.orig	2017-12-04 09:31:58.121936046 -0600
+++ files/Makefile	2017-12-04 09:39:40.553619777 -0600
@@ -103,3 +103,13 @@
 	# @rm -f $(ICP_BUILD_OUTPUT)/ipsec_inline.ko
 	@rm -f *.o *.ko
 
+.PHONY: modules_install
+modules_install:
+	@# Install the external/out-of-tree modules under the
+	@# $(INSTALL_MOD_PATH)/lib/modules/$(KERNELRELEASE)/updates/
+	@# directory in order to supersede any in-kernel modules with the
+	@# same name.  Do not remove the latter modules as that could corrupt
+	@# the package manager (e.g. rpm) database.
+	$(MAKE) -C $(KERNEL_SOURCE_ROOT) M=$(CUR_DIR) INSTALL_MOD_DIR=updates $@
+
+
